test/e2e/declarative_test_fwk/helper.py

```diff
@@ -186,13 +203,32 @@ def _assert_conditions(self, expectations: dict, reference: k8s.CustomResourceRe
             else:
                 raise Exception(f"Condition {condition_name} is provided with invalid value: {expected_value} ")
 
-    def assert_items(self, expectations: dict, state: dict) -> None:
-        """Asserts state against supplied expectations
+    def assert_items_boto3(self, expectations: dict, actual: dict) -> None:
+        """Asserts actual against supplied expectations
+        Override it as needed for custom verifications
```

Comment:
I don't see any override operations in this function

Reply:
I override this function for each resource in `test_scenarios.py` file

Reply:
Update: don't need to override this function anymore.